### PR TITLE
Removing install instructions for Ubuntu 18

### PR DESCRIPTION
**Pull request by dwicinas**
Removed Ubuntu 18 install instructions from all products

Fixes https://enterprisedb.atlassian.net/browse/DF-388

### CODE REVIEW

File: install_template/config.yaml [hunk 1]
```diff
@@ -81,9 +78,6 @@ products:
       - name: Ubuntu 22.04
         arch: x86_64
         supported versions: [55]
-      - name: Ubuntu 20.04
```
Comment (nidhibhammar) [inline install_template/config.yaml:84]: Why Ubuntu 20 is removed from Migration Toolkit?

File: product_docs/docs/migration_toolkit/55/installing/index.mdx [hunk 2]
```diff
@@ -56,7 +56,7 @@ Select a link to access the applicable installation instructions:
 
 ### Debian and derivatives
 
-- [Ubuntu 22.04](linux_x86_64/mtk_ubuntu_22), [Ubuntu 20.04](linux_x86_64/mtk_ubuntu_20)
```
Comment (nidhibhammar) [inline product_docs/docs/migration_toolkit/55/installing/index.mdx:59]: MTK is not supported on Ubuntu 20?

File: product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx [hunk 3]
```diff
@@ -20,6 +20,7 @@ navigation:
   - mtk_sles_12
   - mtk_ubuntu_22
   - mtk_ubuntu_20
+  - mtk_ubuntu_18
```
Comment (nidhibhammar) [inline product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx:23]: Ubuntu 18 is to be removed not added

File: product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx [hunk 4]
```diff
@@ -60,8 +61,6 @@ Operating system-specific install instructions are described in the correspondin
 
 - [Ubuntu 22.04](mtk_ubuntu_22)
 
-- [Ubuntu 20.04](mtk_ubuntu_20)
```
Comment (nidhibhammar) [inline product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx:63]: Need to add this again

File: install_template/config.yaml (hunk 1 above)
Comment (dwicinas) [reply install_template/config.yaml:84]: @nidhibhammar That was a mistake. Should be fixed now.

File: product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx (hunk 4 above)
Comment (dwicinas) [reply product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx]: Fixed

**Comment by dwicinas** [comment]
@nidhibhammar I think I fixed all those issues with Ubuntu 18 and MTK. But here's another question. Is it correct to remove Ubuntu 18 from older versions of EPAS and PEM? No matter what the answer, I have some follow-up work. (Right now it's only half removed.)

File: product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx (hunk 3 above)
Comment (dwicinas) [reply product_docs/docs/migration_toolkit/55/installing/linux_x86_64/index.mdx]: fixed

**Review by nidhibhammar — APPROVED** [review]
LGTM

**Pull request merged**